### PR TITLE
Add json format

### PR DESCRIPTION
**Pull request by jerryliu55**
These are the same changes as in https://github.com/jerryliu55/ruby-bench-web/pull/1

### CODE REVIEW

File: test/integration/repos_test.rb [hunk 1]
```diff
@@ -12,4 +15,34 @@ class ReposTest < ActionDispatch::IntegrationTest
...
+      benchmark_type: benchmark_type,
+      initiator: later_commit
+    )
+
+    get "/#{org.name}/#{repo.name}/commits.json?result_type=#{benchmark_type.category}", 
+      params: { display_count: 2, result_type: benchmark_result_type }
+
+    chart = JSON.parse(response.body, symbolize_names: true)[0]
+    # must have these 4 keys
+    assert ([:benchmark_name, :datapoints, :measurement, :unit] - chart.keys).empty?
```
Comment (jules2689) [inline test/integration/repos_test.rb]:
```ruby
assert_include :benchmark_name, chart.keys
assert_include :datapoints, chart.keys
...
```

seems like a more appropriate way as it requires less thought as to what's going on

File: test/integration/repos_test.rb [hunk 2]
```diff
@@ -12,4 +15,34 @@ class ReposTest < ActionDispatch::IntegrationTest
...
+    benchmark_type = create(:benchmark_type)
+    benchmark_result_type = create(:benchmark_result_type)
+
+    repo = benchmark_type.repo
+    org = repo.organization
+
+    commit = create(:commit, repo: repo)
+    later_commit = create(:commit, repo: repo, created_at: Time.zone.now + 1.day)
+
+    benchmark_run = create(:commit_benchmark_run,
```
Comment (jules2689) [inline test/integration/repos_test.rb]: Do we have fixtures we can use instead?

File: app/controllers/repos_controller.rb [hunk 3]
```diff
@@ -80,6 +99,7 @@ def show_releases
     if (@form_result_type = params[:result_type]) &&
        (@benchmark_type = find_benchmark_type_by_category(@form_result_type))
 
+      versions_calculate_once = ActiveSupport::OrderedHash.new
```
Comment (jules2689) [inline app/controllers/repos_controller.rb]: `versions_calculate_once` should probably be `versions_cache` or something more descriptive. It doesn't actually calculate once

File: test/integration/repos_test.rb (hunk 2 above)
Comment (jerryliu55) [inline test/integration/repos_test.rb]: they use factory girl which is a fixture replacement, so i'm not sure how using fixture here would interact with that (unless we replace everything with fixtures)

**Comment by jerryliu55** [comment]
closing since https://github.com/Shopify/ruby-bench-web/pull/3 is a rewrite and a better solution

**Pull request closed without merging**